### ISSUE TITLE
App doesn't launh

### ISSUE THREAD

**Issue by RadioWavetable** [e1miran/Now-Playing-Serato#6]
Hi,

I'm using a Mac (OSX 10.13.6) and followed the instruction to install and run the app with no luck.
When I try to start the app, nothing happens. I get the security warning (as usual with apps from outside the app store) but when I go passed that, nothing happens. The app doesn't show up anywhere.

Not sure what more info I can give you. 
Please let me know if I can provide you more info to get this issue investigated. I'm more than willing to help.

Thanks in advance and I hope I can get this running!

**Comment by e1miran** [comment]
@RadioWavetable Hi.  Please confirm that you've followed what's mentioned in the install notes here: https://github.com/e1miran/Now-Playing-Serato#important-note

**Comment by RadioWavetable** [comment]
Hi @e1miran !

Thanks for your quick reply. 
I do indeed confirm I have followed those steps.

Thanks for looking into it. It's very much appreciated.

**Comment by e1miran** [comment]
@RadioWavetable Please delete the app from the applications folder.  Then unzip the downloaded zip file again.  This time after moving it to the applications folder and before attempting to open it, run the terminal command: 
`sudo xattr -r -d com.apple.quarantine /path/to/MyApp.app`.  

Make sure you have the correct path to the app.  To get the path, you may be able to right-click the app and copy the path from "Get Info".  Once you run the terminal command, try to open the app.  Also, don't forget to allow the app in the Security & Privacy settings (see image below). Please let me know how it goes.

I really think that this is an issue with the mac operating system quarantining the app since it's unsigned and not from the app store.  When the app is quarantined, the file that the app writes to in order to save its settings is not writeable.  Without being able to open this file, it won't open. Unfortunately there is little I can do about that.

![bypass-gatekeeper-os-x](https://user-images.githubusercontent.com/41588119/95667242-23642200-0b31-11eb-82ba-1df9437b4e33.jpg)

**Comment by e1miran** [comment]
@RadioWavetable were you ever able to open?  I had another user on High Sierra that is having the same issue.

**Comment by RadioWavetable** [comment]
Hey,

No I wasn't able to open it. Tried everything without success.

**Comment by e1miran** [comment]
@RadioWavetable OK.  I'm trying to troubleshoot, but only have Mojave and Catalina available to me.  This seems to be a High Sierra issue.  Again probably having to do with the Apple security.  Sorry you can't open as of yet.

**Comment by RadioWavetable** [comment]
Thanks for looking into it. 
If I can help in any way, please let me know. I don't mind helping testing or troubleshooting.

Thanks in advance for the effort!

**Comment by e1miran** [comment]
@RadioWavetable I managed to install a High Sierra test environment and it looks like by building the app inside of that particular OS now allows it to be opened in High Sierra (weird).  

Please go to the release page and download and test the [High Sierra specific zip package](https://github.com/e1miran/Now-Playing-Serato/releases/tag/v1.3.0).  You will still need to follow the step of copying from your downloads directory to the Applications folder.

You'll probably need to unblock it in System Preferences as well. Please let me know how you make out.
![blocked](https://user-images.githubusercontent.com/41588119/96517294-ae7e9f80-1236-11eb-9534-14142a103521.png)

**Comment by e1miran** [comment]
This is now resolved.  There is a High Sierra specific release for v1.3.0 that addresses it.  Future MAC releases will have it incorporated.

**Comment by RadioWavetable** [comment]
Thanks for your effort! Everything seems to work now as described.
I was able to install the app, following the instructions provided. Now
testing/running it for the first time and all seems to work fine.

Thanks again for this great app!

**Comment by e1miran** [comment]
@RadioWavetable Good to hear.  Hopefully you installed the latest release, v1.4.0.  It has the fix plus some other minor stuff.

**Comment by RadioWavetable** [comment]
Now I did (I was using the 1.3.0 High Sierra version)
Now I'm using the 1.4.0 version and it also works like a charm.

Thanks for this update!

Are you open for suggestions?

**Comment by e1miran** [comment]
> Are you open for suggestions?
Yeah, of course.  If you don't mind, please post a new issue for any feature requests, so I can track them.  Thanks.
